Cargo.toml

```diff
@@ -14,9 +14,9 @@ rust-version = "1.65"
 [features]
 async-std = ["zbus/async-io", "dep:async-std"]
 default = ["async-std"]
-gtk4 = ["gtk4_x11", "gtk4_wayland"]
-gtk4_wayland = ["gdk4wayland", "dep:gtk4"]
-gtk4_x11 = ["gdk4x11", "dep:gtk4"]
+gtk4 = []
```

Comment:
Judging by the build errors, maybe this should be `gtk4 = ["dep:gtk4"]`, and then this dep can be removed from the other two?